app/views/steps/_step.html.erb

```diff
@@ -32,111 +32,102 @@
-              <strong><%= t("protocols.steps.tables") %></strong>
-                <% step.tables.each do |table| %>
-                  <div data-role="hot-table" class="hot-table">
-                    <%= hidden_field(table, :contents, value: table.contents_utf_8, class: "hot-contents") %>
-                    <div data-role="step-hot-table" class="step-result-hot-table"></div>
-                  </div>
-                <% end %>
-            </div>
+          <% if step.description.blank? %>
+            <em><%= t("protocols.steps.no_description") %></em>
+          <% else %>
+            <%= step.description %>
           <% end %>
-          <% assets = ordered_assets(step) %>
-          <% unless assets.blank? then %>
-            <div class="col-xs-12">
-              <strong><%= t("protocols.steps.files") %></strong>
-              <ul>
-                <% assets.each do |asset| %>
-                  <li>
-                    <% if can_view_or_download_step_assets(@protocol) %>
-                      <% if asset.file_present %>
-                        <%= link_to download_asset_path(asset), data: {no_turbolink: true, id: true, status: "asset-present"} do %>
-                          <%= image_tag preview_asset_path(asset) if asset.is_image? %>
-                          <p><%= truncate(asset.file_file_name, length: 50) %></p>
+          <hr>
+          <div class="row">
+            <% unless step.tables.blank? then %>
+              <div class="col-xs-12">
+                <strong><%= t("protocols.steps.tables") %></strong>
+                  <% step.tables.each do |table| %>
+                    <div data-role="hot-table" class="hot-table">
+                      <%= hidden_field(table, :contents, value: table.contents_utf_8, class: "hot-contents") %>
+                      <div data-role="step-hot-table" class="step-result-hot-table"></div>
+                    </div>
+                  <% end %>
+              </div>
+            <% end %>
+            <% assets = ordered_assets(step) %>
+            <% unless assets.blank? then %>
+              <div class="col-xs-12">
+                <strong><%= t("protocols.steps.files") %></strong>
+                <ul>
+                  <% assets.each do |asset| %>
+                    <li>
+                      <% if can_view_or_download_step_assets(@protocol) %>
+                        <% if asset.file_present %>
+                          <%= link_to download_asset_path(asset), data: {no_turbolink: true, id: true, status: "asset-present"} do %>
+                            <%= image_tag preview_asset_path(asset) if asset.is_image? %>
+                            <p><%= truncate(asset.file_file_name, length: 50) %></p>
+                          <% end %>
+                        <% else %>
+                          <%= asset_loading_span(asset) %>
                         <% end %>
                       <% else %>
-                        <%= asset_loading_span(asset) %>
```

Comment:
Why did you remove `asset_loading_span(asset)`? This needs to be here in order for AJAX rendering images to work.